openslides/mediafiles/views.py

```diff
@@ -15,6 +15,13 @@ class MediafileViewSet(ModelViewSet):
     access_permissions = MediafileAccessPermissions()
     queryset = Mediafile.objects.all()
 
+    def get_queryset(self):
```

Comment:
I think you also have to add some logic to the file `access_permissions.py` and use the `check_object_permissions()` method. See agenda access_permissions and views for example: There we have the same case. Some people may see hidden agenda items, others don't. We need this different logic to ensure that only authorized users get this data even via our websocket autoupdate process.